src/FastBoard.cpp

```diff
@@ -19,11 +19,14 @@
 #include "FastBoard.h"
 
 #include <cassert>
+#include <boost/format.hpp>
 #include <array>
 #include <iostream>
 #include <queue>
 #include <sstream>
 #include <string>
+#include <cstdarg>
+#include <cstdio>
```

Comment:
Why would you need these?

Reply:
They are not needed. I will remove them. I must have added them while experimenting with something, then forgot to remove.